index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Twitter Clone</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <!--Grid starts-->
+    <div class="grid">
+      <!--Left Side-Bar Starts-->
+      <div class="left-bar">
+        <!--Twitter Logo on left-bar-->
+        <div class="logo-grid-leftbar">
+          
```

Comment:
I like how clearly you comment your code, makes it nice and easy to follow and is a good practice as things start to scale up.